.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/timothycrosley/isort
+    rev: 5.6.4
+    hooks:
+      - id: isort
+        files: .*.py
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: ["--line-length", "79"]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-ast
```

Comment:
Don't think we need this (slow) one if we are using `black` in its regular mode (it parses into `ast` anyway).

Also I think pytest would bail anyway?

Reply:
Sounds convincing.  Would you agree @sgaist ?

Reply:
Agreed, there might also be other hooks that also uses the AST for their work, so it's one that can be dropped in this case.